project1/css/index.css

```diff
@@ -70,24 +70,28 @@ h1, h2, h3, h4, h5 {
 */
 
 .container {
-  width: 900px;
+  width: 800px;
   margin: 0 auto;
 }
```

Comment:
Using a % is better than setting an exact pixel count. But, I think that was covered the day after you did this. Good job.
Thank you!
Sara Fritz